README.md

```diff
@@ -88,6 +89,34 @@ knp_paginator:
         sortable: '@KnpPaginator/Pagination/sortable_link.html.twig' # sort link template
         filtration: '@KnpPaginator/Pagination/filtration.html.twig'  # filters template
 ```
+#### PHP: (Symfony 5.x)
```

Comment:
Is there a reason to limit this to version 5? Is it not working with Symfony 4 (or 6, in the future)?

Reply:
I understand, I wrote it this way because in the last available LTS version like 4.4, the YAML configuration is still the best practice, however you are right and the user should be able to choose even in those cases.